README.md

```diff
@@ -164,18 +181,24 @@ kubectl delete crd thanosrulers.monitoring.coreos.com
 
 This helmfiles deploys [Hashicorp's Vault](https://www.vaultproject.io/) based on [Bank-Vaults](https://github.com/banzaicloud/bank-vaults). We deploy first the Vault operator, then the following ressources `values/vault.yaml`:
 
-- Vault CRD's, including [Vault policy](https://www.vaultproject.io/docs/concepts/policies) and [Vault authentication](https://www.vaultproject.io/docs/concepts/auth)
+- Vault CRD's, including [Vault policy](https://www.vaultproject.io/docs/concepts/policies), [Vault authentication](https://www.vaultproject.io/docs/concepts/auth), and [Orchestrate Hashicorp Vault Plugin](https://github.com/ConsenSys/orchestrate-hashicorp-vault-plugin)
 
 [Vault policy](https://www.vaultproject.io/docs/concepts/policies)
 ```yaml
   externalConfig:
     policies:
+        {{ if .Environment.Values.metrics.enabled }}
+        - name: prometheus
+          rules: path "sys/metrics" {
+            capabilities = ["list", "read"]
+            }
+        {{ end }}
       - name: allow_secrets
-        rules: path "secret/*" {
+        rules: path "orchestrate/*" {
           capabilities = ["create", "read", "update", "delete", "list"]
```

Comment:
MINOR: Delete is a feature currently not supported by the plugin, so it might be good to remove it from here.

Reply:
Done